app/uk/gov/hmrc/residencenilratebandcalculator/views/not_possible_to_use_service.scala.html

```diff
@@ -0,0 +1,14 @@
+@import uk.gov.hmrc.residencenilratebandcalculator.FrontendAppConfig
+
+@(appConfig: FrontendAppConfig)(implicit request: Request[_], messages: Messages)
+
+@uk.gov.hmrc.residencenilratebandcalculator.views.html.main_template(
+    title = messages("index.browser_title"),
```

Comment:
Presumably there will be different content here ultimately. The message keys for the not possible to use situation should be added to messages and used here.